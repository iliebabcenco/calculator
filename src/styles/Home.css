.home-div {
  width: 75%;
  margin: 0 auto;
  padding: 3rem;
  box-shadow: rgba(100, 100, 111, 0.2) 0 7px 29px 0;
  border-radius: 3px;
}
