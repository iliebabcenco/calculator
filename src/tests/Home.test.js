import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Home from '../components/Home';
import '@testing-library/jest-dom';

it('renders correctly to Home page', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('doesn\'t contain Let’s do some math!', () => {
  render(<Home />);
  expect(screen.queryByText('Let’s do some math!')).not.toBeInTheDocument();
});
