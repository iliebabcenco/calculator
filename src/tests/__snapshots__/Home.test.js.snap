// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly to Home page 1`] = `
<div
  className="home-div"
>
  <h3
    className="home-h3"
  >
    Welcome to our page!
  </h3>
  <p
    className="home-p"
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>
</div>
`;
